File: damhwa_server/spring_server/src/main/java/com/ssafy/damhwa/db/entity/Flower.java
package com.ssafy.damhwa.db.entity;

import lombok.*;

import javax.persistence.*;

@Getter
@Setter
@ToString
@Entity
@NoArgsConstructor
@AllArgsConstructor
public class Flower {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    int fno;

    @Column
    String fname_KR;
    @Column
    String fname_EN;
    @Column
    int fmonth;
    @Column
    int fday;
    @Column
    String flang;
    @Column
    String fcontents;
    @Column
    String fuse;
    @Column
    String fgrow;
    @Column
    String img1;
    @Column
    String img2;
    @Column
    String img3;
    @Column
    String watercolor_img;
    @Transient
    String emotionResult;
}

File: damhwa_client/DamHwa_Android/app/src/main/java/com/example/damhwa_android/data/Flower.kt
package com.example.damhwa_android.data

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Flower(
    @SerializedName("fno")
    val fno: Int,
    @SerializedName("fname_KR")
    val fNameKR: String,
    @SerializedName("fname_EN")
    val fNameEN: String,
    @SerializedName("fmonth")
    val fMonth: Int,
    @SerializedName("fday")
    val fDay: Int,
    @SerializedName("flang")
    val fLang: String,
    @SerializedName("fcontents")
    val fContents: String,
    @SerializedName("fuse")
    val fUse: String,
    @SerializedName("fgrow")
    val fGrow: String,
    @SerializedName("img1")
    val img1: String,
    @SerializedName("img2")
    val img2: String,
    @SerializedName("img3")
    val img3: String,
    @SerializedName("watercolor_img")
    val watercolor_img: String
): Serializable
